File: admin-web/server/src/lib/superAdmin.js
/** Founder / operator emails — protected from demotion & destructive edits. */
export const SUPER_ADMIN_EMAILS = ['christianjoshuacasin@gmail.com'];

export function isSuperAdminEmail(email) {
  if (!email) return false;
  return SUPER_ADMIN_EMAILS.includes(String(email).trim().toLowerCase());
}

File: lib/core/config/super_admin.dart
/// Founder / operator emails that may use both mobile + admin web.
const kSuperAdminEmails = ['christianjoshuacasin@gmail.com'];

bool isSuperAdminEmail(String? email) {
  if (email == null || email.isEmpty) return false;
  return kSuperAdminEmails.contains(email.trim().toLowerCase());
}
